chain/client/src/sync.rs

```diff
@@ -696,6 +699,7 @@ impl StateSync {
                 new_shard_sync_download.downloads[0].run_me = false;
                 new_shard_sync_download.downloads[0].state_requests_count += 1;
                 new_shard_sync_download.downloads[0].last_target = Some(target);
+                *update_sync_status = true;
```

Comment:
How is `update_sync_status` used?

Reply:
It sets StateSyncResult to Changed or Unchanged. The reason is, the shard state isn't changed if we sent no request.